core/src/main/java/io/confluent/kafka/schemaregistry/rest/SchemaRegistryConfig.java

```diff
@@ -395,6 +402,9 @@ public static ConfigDef baseSchemaRegistryConfigDef() {
     .define(KAFKASTORE_TOPIC_CONFIG, ConfigDef.Type.STRING, DEFAULT_KAFKASTORE_TOPIC,
         ConfigDef.Importance.HIGH, KAFKASTORE_TOPIC_DOC
     )
+    .define(KAFKASTORE_TOPIC_VERIFY_CONFIG, ConfigDef.Type.BOOLEAN, DEFAULT_KAFKASTORE_TOPIC_VERIFY,
+        ConfigDef.Importance.HIGH, KAFKASTORE_TOPIC_VERIFY_CONFIG
```

Comment:
Not sure whether this counts as `HIGH` importance!

This config option is important for people trying to deploy to Heroku as if not `schema-registry` fails on boot, but can be safely ignored for deployments running on a Kafka Cluster which they have more control over.